src/index.html

```diff
@@ -28,14 +28,16 @@ <h1>Today's To Do</h1>
             <img src="./assets/return.svg" alt="return icon" class="return" />
           </div>
         </form>
-        <hr />
-        <ul class="todo__list">
-          <!-- Dynamicaly Generated -->
-        </ul>
+        <!-- <hr /> -->
+        <div class="list__container">
+          <ul class="todo__list">
+            <!-- Dynamicaly Generated -->
+          </ul>
+        </div>
       </header>
 
       <div class="footer">
-        <h3>Clear all completed</h3>
+        <h3 class="clearMarked">Clear all completed</h3>
```

Comment:
- [ ][OPTIONAL] This element seems not to work as expected. When clicked, all tasks with the completed status should be deleted from the list. Let's try re-checking this functionality.